src/plccng/scanner/scanner_test.py

```diff
@@ -55,29 +54,39 @@ def test_LexError(errorRaisingMatcher):
     result = list(scanner.scan(oneLineWithError))
     assert isinstance(result[0], LexError)
 
-
-def test_LexError_at_start(errorRaisingMatcher):
+def test_LexError_at_start_goes_through_whole_line_one_character_at_a_time(errorRaisingMatcher):
     scanner = Scanner(errorRaisingMatcher)
     twoLinesWithErrors = parseLines('''\
-first line
-second line
+0123456789
+0123456789
 ''')
     scanner = Scanner(errorRaisingMatcher)
     results = list(scanner.scan(twoLinesWithErrors))
-    assert isinstance(results[0], LexError) and results[0].line.string == 'first line'
-    assert isinstance(results[1], LexError) and results[1].line.string == 'second line'
+    assertResultIsLexErrorAtLineNumberAndColumn(results[0], number=1, column=1)
```

Comment:
I rewrote this test to make it clearer that each LexError advances one character at a time.

But I think we have a problem. The first error should be at column 1 (index 0). But this fails.

Please look into the problem and figure out what's wrong. Is it me? Is it your code? Is it someone else's code?